src/feature/auth/logout/LogoutAction.js

```diff
@@ -29,12 +28,12 @@ export const logout = () => async (dispatch) => {
     const { message } = data;
     localStorage.clear();
     window.location.replace('/');
-    toast.success(message);
     dispatch(logoutSuccess(message));
   } catch (error) {
     const message = (await error.response)
       ? error.response.data.error : 'Something wrong';
-    toast.error(message);
+    localStorage.clear();
+    window.location.replace('/');
```

Comment:
@harerakalex we shouldn't do redirection in action creators. Their purpose is to make dispatch actions only